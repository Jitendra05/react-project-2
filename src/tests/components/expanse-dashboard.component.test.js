import React from 'react';
import ExpanseDashboardComponent from '../../components/expanse-dashboard.component';
import {shallow} from 'enzyme';

test('should render ExpanseDashboardComponent component correctly',()=>{
    const wrapper = shallow(<ExpanseDashboardComponent />);
    expect(wrapper).toMatchSnapshot();
});

