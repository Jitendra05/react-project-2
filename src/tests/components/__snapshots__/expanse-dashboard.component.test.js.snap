// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpanseDashboardComponent component correctly 1`] = `
<div>
  <[object Object] />
  <[object Object] />
  <div
    className="container"
  >
    <[object Object] />
  </div>
</div>
`;
